src/main/scala/sigmastate/eval/RuntimeCosting.scala

```diff
@@ -990,6 +989,11 @@ trait RuntimeCosting extends SigmaLibrary with DataCosting {
 //        val inputC = evalNode(ctx, env, input)
 //        withDefaultSize(inputC.value, inputC.cost + costOf(node))
 
+      case ConstantPlaceholder(_, tpe) =>
+        // todo make a proper Rep type
+        val res = toRep(null)(NothingElement.asElem[Null])
```

Comment:
@ergomorphic For swapping constant placeholders with constants on script deserialization my plan is to use `buildTree`. Seems like we need a rep type for a constant placeholder. Am I on the right track?